examples/android/ReadMe.md

```diff
@@ -1,21 +1,19 @@
-### This example is not fully up to date, please see [our latest example](../DropboxAndroid)
-
 # Example Android application (using the Dropbox SDK for Java.)
 
 This shows the Dropbox API authorization flow and some API calls to retrieve files.
 
 ## Requirements
 
-This example is backwards compatible with Android 4.4 (KitKat). Ensure your build environment supports at least Android SDK version 19.
+This example is backwards compatible with Android 5.0 (Lollipop). Ensure your build environment supports at least Android SDK version 21.
```

Comment:
Did we bump the sample to 21?  We might still be able to support 19, but I propose 21 as well since it is 6 years old at this point.

Reply:
Again, this could be enforced via an Android Manifest.xml that we don't have now.

Reply:
Yes its bumped in the build.gradle [here](https://github.com/dropbox/dropbox-sdk-java/blob/master/examples/android/build.gradle#L10)